modules/ROOT/pages/dataweave-types-coercion.adoc

```diff
@@ -216,6 +216,30 @@ output application/json
 "08/30/2020"
 ----
 
+The following example converts a string to a number with decimal representation. First, you transform the string value `"22"` to `Number` and then to `String` again because `format` adds zeros when converting from a number to a string. If the example string data `"22"` would have been a number you wouldn't need the first `Number` conversion.
```

Comment:
```suggestion
The following example converts a string to a number with decimal representation. First, you transform the string value `"22"` to `Number` and then to `String` again because `format` adds zeros when converting from a number to a string. If the input data is the number `22`, it is not necessary to perform the first `Number` conversion.
```